File: year 1/מבני נתונים/קודים/assignment2/Node.java
package assignment2;

public class Node {
    public Node left, right;
    public Integer data;

    public Node(Integer data) {
        this.data = data;
        this.left = this.right = null;
    }

    public Integer getData() {
        return  this.data;
    }

    public Node getLeft() {
        return  this.left;
    }

    public Node getRight() {
        return this.right;
    }

    public String toString() {
        return "" + this.data;
    }
}

File: year 2/תכנות מערכות ב'/קודים/practice-9-classes/Node.cpp
#include "Node.hpp"
#include <string>

using namespace std;

namespace ariel {
    Node::Node(int data) {
        this->data = data;
        this->next = nullptr;
    }

    int Node::getData() {
        return this->data;
    }

    std::string Node::toString() {
        return to_string(this->data);
    }
}
